recipes/Data-Prep-Kit/intro/.gitignore

```diff
@@ -0,0 +1,20 @@
+## embedded database files
```

Comment:
Alright, to simplify the notebooks,  I am going to create a separate colab friendly version.

I will update the URL references to point to main repo, this will mean the code won't work on colab until merged.